docs/examples.md

```diff
@@ -301,3 +303,20 @@ Common configuration properties that you might want to change from their default
 
 - [cache.max.bytes.buffering](https://kafka.apache.org/documentation/#streamsconfigs):
   The default value in KSQL is `10000000` (~ 10 MB);
+
+
+## Running KSQL
+
+KSQL support various [modes of operation](concepts.md#modes-of-operation), including a standalone mode and a
+client-server mode.
+
+Additionally, you can also instruct KSQL to execute a single statement from the command line.
+
+Example:
+
+```shell
+$ ksql-cli local --exec "SELECT * FROM pageviews LIMIT 5;"
+```
+
```

Comment:
can we add an example of the expected output?

Reply:
Not needed here IMHO.

Reply:
@miguno because it's self-explanatory, or?

Reply:
Because it doesn't add anything to the answer.  The question we want to answer is "how to run a single statement".  The actual KSQL query as well as its output is irrelevant IMHO.